test/simulator/timestamp/src/include/timestamp_manager.h

```diff
@@ -37,7 +37,11 @@ class timestamp_manager {
     public:
     static timestamp_manager &get_timestamp_manager();
     int parse_config(const std::string &, std::map<std::string, std::string> &);
+    uint64_t hex_to_decimal(const std::string &);
+    std::string decimal_to_hex(const u_int64_t);
```

Comment:
Should we be consistent between `u_int64_t ` and `uint64_t`?

Reply:
Good catch!! I will change it to uint64_t

Reply:
Updated in [02cbe29](https://github.com/wiredtiger/wiredtiger/pull/8167/commits/02cbe29df32a4d9abaf9a144dc2ec4fa0a35572e)